syndesis/src/modules/data/DataApp.tsx

```diff
@@ -4,7 +4,7 @@ import VirtualizationCreatePage from './pages/VirtualizationCreatePage';
 import VirtualizationMetricsPage from './pages/VirtualizationMetricsPage';
 import VirtualizationRelationshipPage from './pages/VirtualizationRelationshipPage';
 import VirtualizationsPage from './pages/VirtualizationsPage';
-import VirtualizationSqlQueryPage from './pages/VirtualizationSqlQueryPage';
+import VirtualizationSqlClientPage from './pages/VirtualizationSqlClientPage';
 import VirtualizationViewsPage from './pages/VirtualizationViewsPage';
```

Comment:
Sorry just noticed this. We usually go for named exports only, so we add an `index.ts` file in the folder and have it export all the components from that folder.

The reason for going for named exports over default exports is explained [here](https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/).

One thing that you'll notice if you'll change the code to use named export and the `index.ts` is that you have many interfaces being exported with the same name. That's unfortunate since it leads to uncertainty, and should be fixed.

Reply:
Thanks again @riccardo-forina.. updated accordingly.